optimum/bettertransformer/models/decoder_models.py

```diff
@@ -19,6 +19,7 @@
 from transformers.models.blenderbot.modeling_blenderbot import BlenderbotAttention
 from transformers.models.bloom.modeling_bloom import BloomAttention
 from transformers.models.codegen.modeling_codegen import CodeGenAttention
+from transformers.models.falcon.modeling_falcon import FalconAttention
```

Comment:
Can you add it under a guard with the relevant transformers version? So as to support older transformers versions.

See below:
```python
# TODO: remove once we are much higher than 4.31
if check_if_transformers_greater("4.31"):
    from transformers.models.llama.modeling_llama import LlamaAttention
else:
    from ...utils.dummy_bettertransformer_objects import LlamaAttention
```

Reply:
You will also need to modify the `BACKENDS_MAPPING` in utils/import_utils.py, and add the relevant dummy objects